### DISCUSSION TITLE
Should 1% guideline be revisited in light of vaccine?

### DISCUSSION THREAD

**Discussion by springfielddatarecovery** [microCOVID/microCOVID#934] [Ideas]
From what I understand, the default 1% risk profile of the spreadsheet is based on putting one's yearly risk of disability from COVID on par with their same yearly risk of disability from driving. This was before the vaccine was available, which substantially reduces one's risk of not only testing positive for covid, but having symptoms including long-term disability. You can account for your decreased risk of testing positive for covid by adding your vaccination status, but AFAICT it does not take into account the other protections given by the vaccine. I am vaccinated, so is nearly everybody I know, the tool loses much of it's usefulness now that the consequences of infection are much lower than before.

I have been using this tool for many months and am deeply appreciative of the efforts of the developers, just wanted to start this discussion to see if this is something that would be added or if there's a simple way for me to account for this as a user of the spreadsheet.

**Comment by jy2wong** [comment]
We're discussing and writing this up right now!

There will be a full blog post soon, but for now you might find https://www.mattbell.us/delta-and-long-covid/ interesting (this one gets pretty technical; we're going for a more general audience with the upcoming blog post).

**Reply by springfielddatarecovery** [reply]
Awesome thank you!!

**Reply by blanchardjeremy** [reply]
And here's the blog post: https://www.microcovid.org/blog/delta
